### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by paga16-hash**
working on new MainClass.java and ci.yml

### CODE REVIEW

**Comment by Mala1180** [comment]
Please, solve conflicts

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: ./gradlew clean build
+
+  Completed:
+    needs: [Build]
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: ${{ always() && !contains(join(needs.*.result, ','), 'cancelled') && !contains(join(needs.*.result, ','), 'skipped') }}
+    steps:
+      - run: echo "jobs resulted in ${{ join(needs.*.result, ',') }}"
+      - run: ${{ !contains(join(needs.*.result, ','), 'failure') }}
```
Comment (Mala1180) [inline .github/workflows/ci.yml]: add newline

File: src/main/java/MainClass.java [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (Mala1180) [inline src/main/java/MainClass.java]: Remove the blank line

File: src/main/java/MainClass.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+class MainClass {
+    public static void main(String... args) {
+        System.out.println("Ciao mondo");
```
Comment (Mala1180) [inline src/main/java/MainClass.java]: Write in english please

**Review by Mala1180 — CHANGES_REQUESTED** [review]

**Comment by paga16-hash** [comment]
one thing, the missing blank line at the end of ci.yml is not my work

File: .idea/codeStyles/Project.xml [hunk 4]
```diff
@@ -1,7 +1,43 @@
...
       <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
     </ScalaCodeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </codeStyleSettings>
+=======
+    <ScalaCodeStyleSettings>
+      <option name="MULTILINE_STRING_CLOSING_QUOTES_ON_NEW_LINE" value="true" />
+    </ScalaCodeStyleSettings>
+>>>>>>> 662a67567bad0511650c8e9a55754a5e73bc2094
```
Comment (Mala1180) [inline .idea/codeStyles/Project.xml]: WHAT IS THIS?!

**Review by Mala1180 — CHANGES_REQUESTED** [review]
Please, put .idea files in gitignore except codeStyles

**Pull request merged**